libs/libc/libc.h

```diff
@@ -245,6 +245,13 @@ bool lib_isbasedigit(int ch, int base, FAR int *value);
 
 int lib_checkbase(int base, FAR const char **pptr);
 
+/* Defined in lib_stdio_cb */
+
+ssize_t lib_fread_cb(void *stream, char *buf, size_t size);
+ssize_t lib_fwrite_cb(void *stream, const char *buf, size_t size);
+off_t lib_fseek_cb(void *stream, off_t offset, int whence);
```

Comment:
```suggestion
int lib_fseek_cb(void *stream, off_t offset, int whence);
```

Reply:
fseek for fopencookie is defined with off_t

Reply:
Yes, but fseek could do the conversion, so we don't need seek_internal_cb_t.

Reply:
Yes, but `lib_fseek_cb` passed to `cookie_seek_function_t` which has to be defined with `off_t` to correspond with fopencookie specification.